getting_started/first_3d_game/04.mob_scene.rst

```diff
@@ -164,14 +164,17 @@ between ``-PI / 4`` radians and ``PI / 4`` radians.
         # In this rotation^, the mob will move directly towards the player
         # so we rotate it randomly within range of -90 and +90 degrees.
         rotate_y(randf_range(-PI / 4, PI / 4))
+
  .. code-tab:: csharp
 
-    // We will call this function from the Main scene
+    // This function will be called from the Main scene.
     public void Initialize(Vector3 startPosition, Vector3 playerPosition)
     {
-        // We position the mob and turn it so that it looks at the player.
+        // We position the mob by placing it at startPosition
+        // and rotate it towards playerPosition, so it looks at the player.
```

Comment:
```suggestion
        // and rotating it towards playerPosition, so it looks at the player.
```

Reply:
I don't think `rotating` is correct here. The `and` is connecting the sentences "We position the mob [...]" and "rotate it towards [...]".

I feel like if we changed it to `rotating` then it reads more like "We position the mob by placing it [...]" and "We position the mob by rotating it towards [...]", which is not correct.

Maybe we could add a comma after `startPosition`?

Reply:
Oh, is this supposed to read `(position by placing) and (rotating)` instead of `position by (placing and rotating)`?

Then i would add a comma or maybe even a semicolon and maybe change `and` -> `and then`.

Although this is a comment for just a single function call so my original interpretation of this still seems accurate.

Reply:
> Oh, is this supposed to read `(position by placing) and (rotate it)` instead of `position by (placing and rotating)`?

Yes, I think that's how it's supposed to be read.

> Although this is a comment for just a single function call so my original interpretation of this still seems accurate.

Well, the method `LookAtFromPosition` does both (positions the node and then rotates it):

https://github.com/godotengine/godot/blob/6296b46008fb8d8e5cb9b60af05fa1ea26b8f600/doc/classes/Node3D.xml#L122-L130